apps/arweave/src/ar_data_sync.erl

```diff
@@ -1871,8 +1878,12 @@ get_tx_offset_data_in_range2(TXOffsetIndex, TXIndex, Start, End) ->
 get_tx_data(Start, End, Chunks) when Start >= End ->
 	{ok, iolist_to_binary(Chunks)};
 get_tx_data(Start, End, Chunks) ->
-	case get_chunk(Start + 1, #{ pack => true, packing => unpacked,
-			bucket_based_offset => false }) of
+	{ok, Config} = application:get_env(arweave, config),
+	case get_chunk(Start + 1, #{
+		pack => lists:member(pack_served_chunks, Config#config.enable),
```

Comment:
I couldn't trace through all code paths leading to `get_tx_data`. Some look like they start at an HTTP handler - makes sense that we'd only repack when the option is set for those.

But it looks like a bunch come from `ar_storage:read_tx` which I think is used by a lot of internal processes. I wonder if those processes might depend on the chunk being repacked? Were you able to confirm one way or another?

Reply:
No, but I'll check what I can.

Reply:
worst case you can revert this to `pack => true` - I don't think there's much of a risk there (and probably no new risk?)